c_sharp_json_diff/JsonDiff.cs

```diff
@@ -45,8 +45,11 @@ public JObject NumericDiff(string left, string right)
                     return JObject.Parse(@"{}");
                 }
                 JObject diffJsonObject = JObject.Parse(diff);
-                JObject diffReturnObject = CalculateNumericDiff(diffJsonObject, _isAbsoluteDiff);
-                return diffReturnObject;
+                Dictionary<string, IJsonProcessor<object>> processors = new Dictionary<string, IJsonProcessor<object>>();
+                processors.Add("Int Diff", new IntDiffProcessor());
```

Comment:
Set the ``IntProcessing`` property (proposed in another comment) on the Int/Double processors to ensure that they are processing sensibly. 

For example, an int processor should not fire if the input values are 20 and 25 for left and right because the result would be the fraction 1.25.

For this to work, you may also need to re-order the processors in the list so that the double processor comes first.

Reply:
What is the value in having an int processor at all? An Int can always be cast to a Double, so why not ignore Ints and just deal with doubles? The client can handle rounding the values to Ints if they need to.

Reply:
Good point. I agree that there isn't much use in the Int Diff processor as a Diff processor. If we want Ints, we can just post-process the output of the Double processor using a DoubleToInt diff conversion processor. But we don't need that right now, so it would be fine to just get rid of the Int processor altogether for now.

@puniththota I am actually swaying towards Hector's advice: just get rid of the Int Diff processor & instead of exposing a "numeric diff" function, just expose a "Double Diff" function. Then a large swath of my comments will no longer be relevant and it'll be a lot less work for you.